pkg/util/log/strip.go

```diff
@@ -84,6 +84,16 @@ func matchCert() *regexp.Regexp {
 	)
 }
 
+// AddStrippedKeys allows configuration keys cleaned up
+func AddStrippedKeys(strippedKeys []string) {
+	configReplacer := Replacer{
```

Comment:
We should check that `strippedKeys` is not empty/nil to avoid adding an the empty regexp `()`.